### PR TITLE
1.4 release

### PR DESCRIPTION
**Pull request by Benestar**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Note that for just modifying the docs files you can skip the build using by placing `[skip ci]` in the commit description.

I'd like to have a closer look at this thing in my IDE before release, to check if there is not something else that really makes sense to put in here. Of course if work elsewhere is made significantly easier by already releasing, then perhaps we should just do that, since additional changes can go into a second release soon after without causing issues.

**Comment by JeroenDeDauw** [comment]
Let's finish up #115 first?

**Comment by Benestar** [comment]
@JeroenDeDauw did you find any other issues in your IDE that you want to have in this release? Otherwise I'd say we can do this.

**Comment by JeroenDeDauw** [comment]
Go for it. Several things I want to do, but they can all wait.

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Wikibase DataModel Serialization release notes
 
-## 1.4.0 (dev)
+## 1.4.0 (2015-06-08)
 
 * Added support for showing the component version when loaded via MediaWiki
 * Compatibility with DataModel 3.x added
 * DataModel 3.x is now required
```
Comment (thiemowmde) [inline RELEASE-NOTES.md]: Aren't these two lines redundant?

Comment (Benestar) [inline RELEASE-NOTES.md]: Per https://github.com/wmde/WikibaseDataModelSerialization/blob/master/RELEASE-NOTES.md#110-2014-09-02

**Pull request merged**